### ISSUE TITLE
[WinUI 3 Gallery: Design->Color]: In Dark mode, color contrast ratio for 'Copy brush name' (copy icon) button is not reaching the required contrast ratio 3:1.

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1949]
**Environment Details:**

**OS Version:** 24H2 (OS Build 26100.4315)

**App:** WinUI 3 Gallery

**App Version:** 2.6.0.0 (SDK 1.7)

**AT:** Color Contrast

**Pre-Requisite:**

Settings > Personalization > Color > Dark.

**Repro Steps:**

1. Launch the WinUI 3 Gallery app.

2. Navigate to search box using tab key.

3. Type 'Color' in search box and hit enter.

4. Navigate to 'Color' card and activate it.

5. Navigate to 'Text' list item and select it.

6. Now check color contrast ratio for 'Copy brush name' (icon) button present beside the 'Text /Tertiary' text

**Actual Result:**

In dark mode, color contrast ratio for 'Copy brush name' (copy icon) button is not reaching the required contrast ratio 3:1. We are getting the Color contrast ratio as '2.957:1.

**Note:** Same issue is observed for 'Copy brush name' (copy icon) button present beside the 'Text on Accent /Disabled' text

**Expected Result:**

Color contrast ratio for 'Copy brush name' (copy icon) button should be greater than or equal to 3: 1 in dark mode.

**User Impact:**

Low vision Users will face difficulty in Understanding and accessing the Icon button, if Icon is not reaching the required contrast ratio in dark mode
